.github/workflows/lint-chart.yaml

```diff
@@ -34,13 +35,36 @@ jobs:
         with:
           version: ${{ env.HELM_VERSION }}
 
+      - name: Set up Kustomize
+        uses: syntaqx/setup-kustomize@v1
+        with:
+          kustomize-version: ${{ env.KUSTOMIZE_VERSION }}
+
       - name: Run chart-testing (template)
         run: |
-          helm template --set components={"search,descheduler,schedulerEstimator"} --dependency-update ./charts/karmada --debug > /dev/null
-          helm template --set components={"search,descheduler,schedulerEstimator"},certs.mode=custom --dependency-update ./charts/karmada --debug > /dev/null
-          helm template --set components={"search,descheduler,schedulerEstimator"},installMode=component --dependency-update ./charts/karmada --debug > /dev/null
-          helm template --set installMode=agent --dependency-update ./charts/karmada --debug > /dev/null
-          helm template --dependency-update ./charts/karmada-operator --debug > /dev/null   
+          cat <<EOF > post-render.sh
+          #!/bin/sh
+          cat > all.yaml
```

Comment:
so you build a empty file named `all.yaml`? and then when executing `kustomize` it will build all yaml from `template` dirtory?

Reply:
Nope, this is used in `helm --post-render`, so it's a pipe, `all.yaml` contains helm stdout contents, this makes `kustomize` able to build all template files.

Reply:
Thank you for your reply

I see you added a line of explanation in the comment, this makes it easier to understand. 👍